src/test/java/de/uni_hamburg/informatik/swt/se2/mediathek/services/verleih/VerleihServiceImplTest.java

```diff
@@ -55,8 +58,13 @@ public VerleihServiceImplTest()
         medium = new CD("CD4", "baz", "foo", 123);
         medienbestand.fuegeMediumEin(medium);
         _medienListe = medienbestand.getMedien();
+        
```

Comment:
Das wird so nicht klappen, in Zeile 59 gibst du die momentane Referenz an VerleihServiceImpl weiter, jedoch setzt du in Zeile _vormerkService auf eine neue Instanz, _service zeigt jedoch noch auf die alte!
Das ist so wie
x = new FalschesX()
y = new Y(x)
x = new RichtigesX()

y zeigt immer noch auf das falsche X!